cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/template-control.tsx

```diff
@@ -25,119 +25,125 @@ export interface TemplateControlProps {
 export const TemplateControl: FC<TemplateControlProps> = ({
     canvasInstance,
     isDrawing,
-    jobInstance,
     frame,
     labels,
-    onCreateAnnotationsAndGrouping
 }) => {
     const template: VertexTemplate[] = [
         {
             location: [0.5, 0],
-            nameHint: "Head"
+            nameHint: 'Head',
         },
         {
             location: [0.5, 0.1],
-            nameHint: "Neck"
+            nameHint: 'Neck',
         },
 
         {
             location: [0.28, 0.12],
-            nameHint: "RShoulder"
+            nameHint: 'RShoulder',
         },
         {
             location: [0.26, 0.3],
-            nameHint: "RElbow"
+            nameHint: 'RElbow',
         },
         {
             location: [0.24, 0.5],
-            nameHint: "RWrist"
+            nameHint: 'RWrist',
         },
 
         {
             location: [0.72, 0.12],
-            nameHint: "LShoulder"
+            nameHint: 'LShoulder',
         },
         {
             location: [0.75, 0.3],
-            nameHint: "LElbow"
+            nameHint: 'LElbow',
         },
         {
             location: [0.76, 0.5],
-            nameHint: "LWrist"
+            nameHint: 'LWrist',
         },
 
         {
             location: [0.30, 0.45],
-            nameHint: "RHip",
+            nameHint: 'RHip',
         },
         {
             location: [0.27, 0.75],
-            nameHint: "RKnee",
+            nameHint: 'RKnee',
         },
         {
             location: [0.25, 1.0],
-            nameHint: "RAnkle",
+            nameHint: 'RAnkle',
         },
 
         {
             location: [0.70, 0.45],
-            nameHint: "LHip",
+            nameHint: 'LHip',
         },
         {
             location: [0.73, 0.75],
-            nameHint: "LKnee",
+            nameHint: 'LKnee',
         },
         {
             location: [0.75, 1.0],
-            nameHint: "LAnkle",
+            nameHint: 'LAnkle',
         },
 
         // {
         //     location: [0, 0],
-        //     nameHint: "tl",
+        //     nameHint: 'tl',
         // },
         // {
         //     location: [1, 0],
-        //     nameHint: "tr",
+        //     nameHint: 'tr',
         // },
         // {
         //     location: [0, 1],
-        //     nameHint: "bl",
+        //     nameHint: 'bl',
         // },
         // {
         //     location: [1, 1],
-        //     nameHint: "br",
+        //     nameHint: 'br',
         // }
     ];
 
     const [points, setPoints] = useState<(PointData & { id: number })[]>([]);
 
+    const getLabel = (nameHint: string | undefined) => {
+        const defaultLabel = labels[0] || { id: 1 };
+
+        if (nameHint === undefined) return defaultLabel;
+
+        const nameHintLowerCase = nameHint.toLowerCase();
+        const alternatives = labels.filter((name) => name.includes(nameHintLowerCase));
```

Comment:
I do not understand this: it looks like labels is assumed to be a list of names/strings..?